pallets/authors-manager/src/lib.rs

```diff
@@ -390,6 +407,35 @@ impl<T: Config> Pallet<T> {
         Ok(())
     }
 
+    fn publish_to_bridge(
+        eth_public_key: &ecdsa::Public,
+        author_id: &T::AccountId,
+        operation: AuthorOperationTier1Endpoint,
+    ) -> Result<u32, DispatchError> {
+        let decompressed_eth_public_key =
+            decompress_eth_public_key(*eth_public_key).map_err(|_| Error::<T>::InvalidPublicKey)?;
+
+        let author_id_bytes = <T as pallet::Config>::AccountToBytesConvert::into_bytes(author_id);
+
+        let params = match operation {
+            AuthorOperationTier1Endpoint::Remove => vec![
+                (b"bytes32".to_vec(), author_id_bytes.to_vec()),
+                (b"bytes".to_vec(), decompressed_eth_public_key.to_fixed_bytes().to_vec()),
+            ],
+            AuthorOperationTier1Endpoint::Add => vec![
+                (b"bytes".to_vec(), decompressed_eth_public_key.to_fixed_bytes().to_vec()),
+                (b"bytes32".to_vec(), author_id_bytes.to_vec()),
+            ],
```

Comment:
this can be improve, by having a function in the enumeration, say `convert_to_params_data` ,that takes these two arguments and returns the bytes.
so the code would become, `operation.convert_to_params_data(author_id_bytes, decompressed_eth_public_key)`